packages/ui-utils/src/call-chat-api.ts

```diff
@@ -106,3 +106,91 @@ export async function callChatApi({
     }
   }
 }
+
+export async function resumeChatApi({
```

Comment:
Can this be unified with the `processDataStream` function instead by adding a param to `processDataStream`?

Reply:
done! added `requestType` param to differentiate between `generate` and `resume` call.